### DISCUSSION TITLE
slide_kitsune (slide and surfing map for NZP) [FTE/PC ONLY]

### DISCUSSION THREAD

**Discussion by bigjackson** [nzp-team/nzportable#706] [Map Releases]
**slide_kitsune is a NZP map that uses map geometry in most surf maps in GoldSrc/Source. This map allows you to slide down ramps and surf them.**

![image](https://github.com/nzp-team/nzportable/assets/60328163/281e6ad1-ea63-4a9a-b276-e50bacdf8602)
![image](https://github.com/nzp-team/nzportable/assets/60328163/92a41463-9609-49f3-908b-78b6e59cada5)

**Comment by StickyPuddingg** [comment]
Looks really cool ill give it a try later today, thank you for your contribution to the nzp community!

**Comment by dogerizergaming** [comment]
The link is dead, you should reupload it ;)

**Reply by immortalshortie** [reply]
ong

**Reply by bigjackson** [reply]
the link works now

**Reply by Peter0x44** [reply]
@bigjackson clicking the link directly here doesn't work, but pasting it in discord message and clicking it does. I think you should use google drive or github itself for something more persistent.

**Reply by bigjackson** [reply]
@Peter0x44 alright, i'll try that. thank you.

**Reply by bigjackson** [reply]
changed the LINK now it should actually finally work

**Reply by Peter0x44** [reply]
Can confirm it works.

**Comment by KingDoge176** [comment]
How do you get past the green hole

**Reply by KingDoge176** [reply]
At the start
